README.md

```diff
@@ -68,6 +69,21 @@ $ sudo find / -name qmake
 ## Run:
 ``` ./start.sh [options] [script file name] ```
 
+## Documentation generation
+### Download and install [ldoc](stevedonovan.github.io/ldoc/manual/doc.md.html)
+```
+sudo apt install luarocks
+sudo luarocks install penlight
```

Comment:
@pestOO The problem is in `sudo apt install luarocks`. On ubuntu 14.04 it installed version 2.0.8, which has a problem, that was fixed in newer version.